Paper.tex

```diff
@@ -1661,6 +1661,7 @@ \section{Signing Transactions}\label{app:signing}
 We assume the existence of functions $\mathtt{\small ECDSAPUBKEY}$, $\mathtt{\small ECDSASIGN}$ and $\mathtt{\small ECDSARECOVER}$. These are formally defined in the literature, \eg by \cite{ECDSAcerticom}.
 \begin{eqnarray}
 \mathtt{\small ECDSAPUBKEY}(p_{\mathrm{r}} \in \mathbb{B}_{32}) & \equiv & p_{\mathrm{u}} \in \mathbb{B}_{64} \\
+\ e & \equiv & \hyperlink{h T}{h(T)}\\
```

Comment:
Where is this newly defined variable `e` used?

Reply:
For clarity, but the label needs updating as it has a space.

Reply:
It is defined because it is used as an input to ECDSASIGN and ECDSARECOVER.